frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,54 @@
+.input {
+  padding: 20px 24px;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #000;
+  border: 1px solid #9f18d9;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #9f18d9 0.5;
```

Comment:
This declaration doesn't work.
![Uploading Screenshot 2022-08-02 at 08.48.55.png…]()

Reply:
This declaration doesn't work.

<img width="479" alt="Screenshot 2022-08-02 at 08 51 59" src="https://user-images.githubusercontent.com/26620750/182310646-cca003e8-0907-4467-87fc-b64446f5f714.png">